### PR TITLE
Add Network Eightbox Testnet

### PR DESCRIPTION
**Pull request by eightbox**
Add Network Eightbox Testnet

### CODE REVIEW

File: _data/icons/eightbox.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+      "url": "ipfs://QmQ516hXZRyoNCJ7mXwG4fN4ECQRvkB3EmDpyBR8GTdxgR",
+      "width": 200,
+      "height": 200,
+      "format": "png"
+    }
+  ]
+
```
Comment (ligi) [inline _data/icons/eightbox.json:9]:
```suggestion
```

**Review by ligi — CHANGES_REQUESTED** [review]
tl;dr: it is now required to put the binary icon into iconDownload

why: it was a PITA to always `ipfs get` in the review process - sometimes it takes *really long* or even fails (especially with pinata) - this took significant time (which adds up with the amount of chains added) and slowed down the review process. Also sometimes CIDs become unavailable after - big mess. Hence it is now required to put them in iconDownload (still named via CID)

**Comment by eightbox** [comment]
@ligi I've got the CID file, but I don't know how to download it. I'm using webui ipfs
image

**Comment by eightbox** [comment]
I've uploaded it on the pinata but the status is searching

image

File: _data/icons/eightbox.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "https://ipfs.eightbox.org/QmQ516hXZRyoNCJ7mXwG4fN4ECQRvkB3EmDpyBR8GTdxgR",
```
Comment (ligi) [inline _data/icons/eightbox.json:3]:
```suggestion
    "url": "ipfs://QmQ516hXZRyoNCJ7mXwG4fN4ECQRvkB3EmDpyBR8GTdxgR",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by eightbox** [comment]
why does it always fail? is there any solution?

**Comment by ligi** [comment]
The image you uploaded is not a image ..

**Comment by eightbox** [comment]
> The image you uploaded is not a image ..

Then can you help me? because I've followed various ways on Google, but haven't found it

**Comment by ligi** [comment]
send me the icon and 100DAI - I can't help random chains for free - the day only has 24h

**Review by Aqeebrao — APPROVED** [review]

**Review by Aqeebrao — APPROVED** [review]
#2101

**Pull request closed without merging**